dev-docs/Cloud-Env-Testing.md

```diff
@@ -47,14 +47,17 @@ Follow these steps to run the workflow:
       e.g. `elastic / unified-release - staging # 8.9 - 11 - 8.9.0-c6bb8f7a Success after 4 hr 58 min`. Now just copy it
       and replace it the image path: `docker.elastic.co/cloud-release/elastic-agent-cloud:8.9.0-c6bb8f7a`.
 
+    - `add-d4c-integration` (optional): Set this to true to enable the Defend For Containers (D4C) integration. When enabled, the D4C integration will be installed alongside the KSPM EKS integration under the same agent policy.
```

Comment:
why not always have it as part of the cloud environment?

Reply:
Our cloud environment deployment is designed to encompass all our integrations by default, mainly for the purposes of testing and exploration. The introduction of Defend for Containers introduces an additional layer of configuration that may be necessary when engaging with customers in real-world scenarios. As we currently address scenarios where both integrations coexist on the same agent, enabling Defend for Containers becomes an optional configuration. The primary rationale is to provide flexibility and address any compatibility concerns that might arise. By default, our focus is on ensuring a clean deployment with CSP products.

Reply:
C4D lies under our responsibilities, no different than any other integration we have as part of this workflow

Reply:
Implementation follows the task's plan. If C4D needs permanent deployment, I'll adjust the workflow to remove the enable/disable option.